test/Atmos/EDMF/ekman_layer.jl

```diff
@@ -85,6 +85,48 @@ function init_state_prognostic!(
     return nothing
 end;
 
+"""
+    subdomain_surface_values(
+        atmos::AtmosModel{FT},
+        state::Vars,
+        aux::Vars,
+        zLL::FT,
+    ) where {FT}
+
+Returns the surface value of EDMF variables for the
+Ekman layer simulation, overriding general method.
+"""
+function subdomain_surface_values(
```

Comment:
Should we define this next to the existing one and just define and use a new surface model? Simply overriding the existing method seems risky (and will require more work to merge later when we want to move all of this to `src/`).

Reply:
Done!